js/build.js

```diff
@@ -136,11 +157,13 @@ Fliplet.Widget.instance('chat', function (data) {
   });
 
   $wrapper.on('click', '.chat-text', function() {
-    getElemHandler($(this));
-    $(this).tooltip('toggle');
-    $(this).parents('.chat-body').toggleClass('selected');
-    $(this).parents('.chats').find('.chat-text[aria-describedby]').not(this).parents('.chat-body').removeClass('selected');
-    $(this).parents('.chats').find('.chat-text[aria-describedby]').not(this).tooltip('hide');
+    if (Fliplet.Env.get('platform') === 'native') {
+      getElemHandler($(this));
+      $(this).tooltip('toggle');
+      $(this).parents('.chat-body').toggleClass('selected');
+      $(this).parents('.chats').find('.chat-text[aria-describedby]').not(this).parents('.chat-body').removeClass('selected');
+      $(this).parents('.chats').find('.chat-text[aria-describedby]').not(this).tooltip('hide');
```

Comment:
Instead of typing the entire line of chained jQuery statements, you can use `removeClass('selected').end().tooltip('hide');`